tools/dep_updaters/update-inspector-protocol.sh

```diff
@@ -37,7 +37,7 @@ python "$BASE_DIR/tools/inspector_protocol/roll.py" \
   --node_src_downstream "$BASE_DIR" \
   --force
 
-NEW_VERSION=$(grep "Revision:" "$DEPS_DIR/inspector_protocol/README.node" | sed -n "s/^Revision: \(\\w*\)/\1/p")
+NEW_VERSION=$(grep "Revision:" "$DEPS_DIR/inspector_protocol/README.node" | sed -n "s/^Revision: \(\\w*\)/\1/p" | head -c 7)
```

Comment:
7 is too small IMO, https://github.com/nodejs/node/commit/b0e1ef198b170a8e8df17347783a0e31a1a737c8 finds a collision with an unrelated commit. Let's use the max length that keeps us under the 72 char limit:
```suggestion
NEW_VERSION=$(grep "Revision:" "$DEPS_DIR/inspector_protocol/README.node" | sed -n "s/^Revision: \(\\w*\)/\1/p" | head -c 36)
```